### DISCUSSION TITLE
Replace Lena test image

### DISCUSSION THREAD

**Discussion by agoose77** [alandefreitas/matplotplusplus#113] [Ideas]
Apologies if this isn't the best mechanism by which to open this discussion.

The Lena test image has been used for quite some time in evaluating image processing algorithms. Whilst I recognise that it has been useful to have an (informally) standardised sample, I would like to propose that we replace it here in favour of an alternative (see referenced journal article).

Several journals such as Nature Nanotechnology ([A note on the Lena image. Nature Nanotech 13, 1087 (2018).](https://doi.org/10.1038/s41565-018-0337-2)) no longer accept submissions containing this image. The motivation is relatively transparent; in the contemporary landscape of egalitarian and inclusive collaborations, the Lena image is outmoded and un-reflective of these values. 

Given the precedent set by the academic platform, I hope that it is not difficult to acknowledge the merit and importance of making the same change here.

Thanks all!

**Comment by alandefreitas** [comment]
Nice point. We'd have to have a look at the [alternatives](https://www.tandfonline.com/doi/full/10.1080/09500340.2016.1270881) then. I'm converting this into a discussion until we decide which image to use, so it's easier for people to contribute with their comments. 
Once we decide which image to use, we can reference the discussion in a new issue as a feature request. 

From the perspective of optics, the new image should pass the criteria in the [alternatives](https://www.tandfonline.com/doi/full/10.1080/09500340.2016.1270881) above. From the perspective of the algorithms, it should be colored (not just formally, but lots of colors), and preferably have a face on it, so we can better understand the result of the image transformation algorithms.
